agent/xds/delta.go

```diff
@@ -409,7 +411,18 @@ func (s *Server) applyEnvoyExtensions(resources *xdscommon.IndexedResources, cfg
 				"partition", cfg.ServiceName.Partition,
 			}
 
+			getMetricLabels := func(err error) []metrics.Label {
```

Comment:
Do you think it makes sense to include `namespace` and `partition` here too?

Reply:
When running in k8s, it generates output like this usually, which I figured was sufficient because it includes the namespace. I should add those extra fields just to be safe, though.

```
consul_envoy_extension_extend{app="consul", chart="consul-helm", component="server", controller_revision_hash="consul-server-59b6f98bb4", error="false", extension="builtin/lua", hasDNS="true", instance="10.42.0.13:8500", job="kubernetes-pods", kubernetes_namespace="consul", kubernetes_pod_name="consul-server-0", quantile="0.5", release="consul", service="forward1-sidecar-proxy", statefulset_kubernetes_io_pod_name="consul-server-0", version="builtin/1.15.0"}
```